README.md

```diff
@@ -9,6 +9,8 @@ for [iOS](https://itunes.apple.com/app/id510623322) and
 
 This repository contains submodules. Clone it with `git clone --recursive`. If you forgot,
 run `git submodule update --init --recursive`.
+Please due to some hard coded paths for development environment do not use custom names
+for a repository root directory. At the moment it MUST remain "omim".
```

Comment:
Even if there is a problem, it has nothing to do with submodules and even with README.
A guarding sentence like 'This document assumes that you have downloaded the repository to a folder named omim' may indeed be added to INSTALL or CONTRIBUTING if @Zverik approves.